lib/order.rb

```diff
@@ -0,0 +1,18 @@
+require './lib/menu.rb'
+
+class Order
+	attr_reader :basket, :total
+
+	def initialize(menu = Menu.new)
+		@basket = []
+		@menu = menu
+	end
+
+	def add(dish_number)
+		@menu.dishes.each { |item| @basket << item if item[:dish_number] == dish_number }
```

Comment:
I did something similar here. I think a good next step for both of us would be to get it to raise an error if the dish doesn't exist in the menu.